code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -43,15 +43,58 @@
 		/obj/item/weapon/card/id
 		)
 
+//Med Borg Medical Gripper
+/obj/item/weapon/gripper/medical
+	name = "Medical Gripper"
+	desc = "A specialized manipulator for organs, iv-bags, pills, patches, viral samples, vials, and other medical assets."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "medgripper"
+
+	//Has a list of items that it can hold.
+	can_hold = list(
+		/obj/item/organ, //surgery
+		/obj/item/weapon/reagent_containers/blood, //i-v bags for surgery
+		/obj/item/weapon/reagent_containers/pill, //pills & patches for treatment
+		/obj/item/weapon/reagent_containers/food, //treatment & monkey cubes
+		/obj/item/weapon/reagent_containers/glass/beaker/vial, //virology
+		/obj/item/weapon/dnainjector, //genetics; for applying clean injectors
+		/obj/item/clothing/mask/breath, //breath masks for anesthetic
+		/obj/item/weapon/tank, //tanks handling to apply oxygen and anesthetic to patients
+		/obj/item/weapon/disk, //virology & genetics
+		/obj/item/weapon/implant, //surgery
+		/obj/item/weapon/virusdish, //virology
+		/obj/item/weapon/storage/box,//Medical supply retrieval; pills, bodybags, monkey cubes for genetics/virology.
+		/obj/item/weapon/storage/firstaid, //Medical supply retrieval.
+		/obj/item/stack/sheet/biomass, //biomass; mainly for supplying the cloner
+		/obj/item/bodybag //morgue
+		)
+
 /obj/item/weapon/gripper/attack_self(mob/user as mob)
-	if(wrapped)
+	if(istype(wrapped,/obj/item/weapon/storage))//Allow grippers that can carry containers to empty them
+		var/obj/item/weapon/storage/W = wrapped
+		if(W.contents.len >= 1)
+			var/response = sd_Alert(user,"Empty this container?","Empty Container?",list("Yes","No"))
```

Comment:
Suprising, since it's ridiculously old.
Still, use alert, it has no place being here.